packages/components/src/test/properties/inputs/UniquePropertyValuesSelector.test.tsx

```diff
@@ -933,13 +933,16 @@ describe("UniquePropertyValuesSelector", () => {
       await user.type(searchSelector!, "Searched");
 
       // ensure all searched for values are shown
-      await waitFor(() => {
-        expect(queryByText("SearchedValue1")).to.not.be.null;
-        expect(queryByText("SearchedValue2")).to.not.be.null;
-        expect(queryByText("SearchedValue3")).to.not.be.null;
-        expect(queryAllByText(/SkippedValue/)).to.be.empty;
-      });
-      expect(getDistinctValuesIteratorStub).to.be.calledThrice;
+      await waitFor(
+        () => {
+          expect(getDistinctValuesIteratorStub).to.be.calledThrice;
+          expect(queryByText("SearchedValue1")).to.not.be.null;
+          expect(queryByText("SearchedValue2")).to.not.be.null;
+          expect(queryByText("SearchedValue3")).to.not.be.null;
+          expect(queryAllByText(/SkippedValue/)).to.be.empty;
+        },
+        { timeout: 2000 },
```

Comment:
Why is this needed?

Reply:
This test was occasionally failing. Did some debugging and it seems like the pages are requested correctly but sometimes it timeouts before the last page loaded.